Source/WebKit/WebProcess/WebCoreSupport/WebChromeClient.cpp

```diff
@@ -991,7 +1018,7 @@ void WebChromeClient::getBarcodeDetectorSupportedFormats(CompletionHandler<void(
 RefPtr<WebCore::ShapeDetection::FaceDetector> WebChromeClient::createFaceDetector(const WebCore::ShapeDetection::FaceDetectorOptions& faceDetectorOptions) const
 {
 #if ENABLE(GPU_PROCESS)
-    auto& remoteRenderingBackendProxy = m_page.ensureRemoteRenderingBackendProxy();
+    auto& remoteRenderingBackendProxy = protectedPage()->ensureRemoteRenderingBackendProxy();
```

Comment:
This doesn't seem right. Why is it safe to hold the reference from this function without protecting the page in a local variable?